File: 2011-final-value-of-variable-after-performing-operations/2011-final-value-of-variable-after-performing-operations.kt
class Solution {
fun finalValueAfterOperations(operations: Array<String>): Int {
    var num = 0
    operations.forEach {
        if (it.contains("++")){
            num++
        }
        else if (it.contains("--")){
            num--
        }
    }
    return num
}

}

File: 2011-final-value-of-variable-after-performing-operations/2011-final-value-of-variable-after-performing-operations.js
/**
 * @param {string[]} operations
 * @return {number}
 */
var finalValueAfterOperations = function(operations) {
    let X=0
    operations.forEach((v)=>{
        if(v[0]==="-" || v[1]==="-") X-=1
        else X+=1
    })
    return X
};